scripts/run_linter.sh

```diff
@@ -79,25 +79,18 @@ $PYTHON_CMD $TOOLS_DIR/pycodestyle-2.3.1/pycodestyle.py -v || exit 1
 # installation.
 source $(dirname $0)/setup_node_modules_util.sh
 
-install_node_module @mapbox/stylelint-processor-arbitrary-tags 0.2.0
-install_node_module postcss 5.0.0
-install_node_module postcss-syntax 0.10.0
-install_node_module stylelint 9.2.1
-install_node_module stylelint-config-standard 18.2.0
-
-if [ ! -f $FOUNDATION_DIR/static/assets/css/stylesheet.css ]; then
+export CSS_STYLESHEET=$APP_DIR/static/assets/css/stylesheet.css
+if [ ! -f $CSS_STYLESHEET ]; then
   echo "stylesheet.css does not exist!"
   exit 1
 fi
 
 echo Running stylelint on all CSS files
-$NODE_MODULE_DIR/stylelint/bin/stylelint.js "static/assets/css/stylesheet.css" --config=$FOUNDATION_DIR/.stylelintrc || exit 1
+$NODE_MODULE_DIR/stylelint/bin/stylelint.js $CSS_STYLESHEET --config=$FOUNDATION_DIR/.stylelintrc --formatter verbose || exit 1
 
 echo Running stylelint on all HTML files
-$NODE_MODULE_DIR/stylelint/bin/stylelint.js "static/pages/**/*.html" --config=$FOUNDATION_DIR/.stylelintrc || exit 1
-
-install_node_module htmllint 0.7.2
-install_node_module htmllint-cli 0.0.7
+$NODE_MODULE_DIR/stylelint/bin/stylelint.js "app/static/pages/**/*.html" --config=$FOUNDATION_DIR/.stylelintrc --formatter verbose || exit 1
 
-echo Running htmllint on all HTML files
-$NODE_MODULE_DIR/htmllint-cli/bin/cli.js --rc=$FOUNDATION_DIR/.htmllintrc || exit 1
+echo Running htmllint on HTML files
+find $APP_DIR -name '*.html'
```

Comment:
Hm, what does this line do?

Reply:
https://travis-ci.com/oppia/foundation-website/jobs/147387957#L2849
This command is used to find all HTML files in the `/app` folder. The results are then piped into htmllint-cli linter.

htmllint-cli does not have a verbose flag so I was simply trying to help with the verbosity, instead of just printing:
`
[htmllint] found 0 errors out of 15 files
`
All 15 HTML filenames are printed ahead of time before the HTML linter makes a pass over each HTML files and then print the above statement.

Reply:
Ah, but isn't this command included as part of the next line already? Sorry if I misunderstood.

Reply:
Ah, I see now, sorry. Thanks!

Also, apologies, I think I introduced a merge conflict when merging in the other PR. Could you fix that? Once that's done feel free to merge this.

Thanks!